modules/ipi-install-configuring-the-bios.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-the-bios_{context}"]
+= (Optional) Configuring the BIOS
+
+The following procedure configures the BIOS during the installation process.
+
+.Procedure
+. Create the manifests.
+
+. Modify the `BareMetalHost` resource file corresponding to the node:
++
+[source,terminal]
+----
+$ vim clusterconfigs/openshift/99_openshift-cluster-api_hosts-*.yaml
+----
+
+. Add the BIOS configuration to the `spec` section of the `BareMetalHost` resource:
++
+[source,terminal]
+----
+spec:
+  firmware:
+    simultaneousMultithreadingEnabled: true
+    sriovEnabled: true
+    virtualizationEnabled: true
+----
++
+[NOTE]
+====
+Red Hat supports three BIOS configurations. See the link:https://docs.openshift.com/container-platform/{product-version}/post_installation_configuration/bare-metal-configuration.html[Bare metal configuration] for details. Only servers with BMC type `irmc` are supported. Other types of servers are currently not supported.
```

Comment:
This link isn't rendering.

![image](https://user-images.githubusercontent.com/7443887/169897596-bbf2102c-c8d7-45d2-b97c-74b42910c06c.png)

It might be because of trying to use the {product-version} in the URL, but still, linking to docs.openshift.com isn't something that we want to allow. Because these docs are also on the portal, so it wouldn't be a good experience to redirect users viewing portal docs to docs.openshift.com.

The workaround for when you want to add an xref in a module and can't is to add the `.Additional resources` section in the assembly right after the module inclusion, with the xref.

Reply:
Fixed.